src/coreclr/jit/stacklevelsetter.cpp

```diff
@@ -19,8 +19,12 @@ StackLevelSetter::StackLevelSetter(Compiler* compiler)
     , throwHelperBlocksUsed(comp->fgUseThrowHelperBlocks() && comp->compUsesThrowHelper)
 #endif // !FEATURE_FIXED_OUT_ARGS
 {
+#ifndef TARGET_WASM
     // The constructor reads this value to skip iterations that could set it if it is already set.
     compiler->codeGen->resetWritePhaseForFramePointerRequired();
+#else
+    assert(false); // Wasm - TODO can this be ignored?
```

Comment:
It should be possible to exclude the whole StackLevelSetter

Reply:
Thanks, yes, removed also.